aiidalab_widgets_base/viewers.py

```diff
@@ -71,30 +84,182 @@ class StructureDataViewer(ipw.VBox):
     :param downloadable: If True, add link/button to download the content of the object
     :type downloadable: bool"""
 
-    def __init__(self, structure=None, downloadable=True, **kwargs):
+    SELECTION_SIZE = 10
```

Comment:
I think a traitlet or a member attribute would be more appropriate for this.

Reply:
I am not sure here. `SELECTION_SIZE` is the size of a ball that appears when you select an atom, I don't think it should be defined per member. Traitlet doesn't make sense since I do not expect to call a function when it gets modified.

Reply:
Ok, I renamed it to avoid confusions.